src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralConfiguredBrowseObject.xaml

```diff
@@ -57,4 +57,5 @@
   <StringProperty Name="DebugSymbols" DisplayName="Debug symbols" Visible="False"/>
   <EnumProperty Name="FileAlignment" DisplayName="File Alignment" Visible="False"/>
   <StringProperty Name="BaseAddress" DisplayName="Base address" Visible="False"/>
+  <StringProperty Name="FullPath" DisplayName="Project Folder" ReadOnly="True"/>
```

Comment:
yes , looks like a special case it picks up the relative location to the projectpath [here](https://github.com/dotnet/roslyn-project-system/blob/master/src/Microsoft.VisualStudio.AppDesigner/PropPages/PropPageUserControlBase.vb#L3349) and also see it defined in the [generalbrowseobject](https://github.com/dotnet/roslyn-project-system/blob/9a7876305be935d0605a492bb1583421bd0e0a2e/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml#L79)